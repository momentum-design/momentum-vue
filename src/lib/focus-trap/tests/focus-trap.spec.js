import { shallowMount } from '@vue/test-utils';
import FocusTrap from '../index.vue';

describe('FocusTrap', () => {
  it('should match snapshot', () => {
    const wrapper = shallowMount(FocusTrap, {
      propsData: {
        focusTrapOptions: {
          initialFocus: 'body'
        }
      },
      slots: {
        default: '<div />'
      }
    });
    expect(wrapper.html()).toMatchSnapshot();
  });
});

