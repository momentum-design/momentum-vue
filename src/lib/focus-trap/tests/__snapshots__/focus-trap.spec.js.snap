// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FocusTrap should match snapshot 1`] = `<div></div>`;
